### PR TITLE
Registering all the parameters to the optimizer

### PR DESCRIPTION
**Pull request by fgiobergia**
The tutorial states the following:
> Notice **although we register all the parameters in the optimizer**, the only parameters that are computing gradients (and hence updated in gradient descent) are the weights and bias of the classifier.

However, at the moment, only the classifier (`model.fc`)'s parameters are registered in the optimizer. With this (minor) fix, the code should be in agreement with the text.

### CODE REVIEW

**Comment by holly1238** [comment]
Closing this PR. The latest version of the tutorial is already updated with this fix: https://pytorch.org/tutorials/beginner/blitz/autograd_tutorial.html#sphx-glr-beginner-blitz-autograd-tutorial-py
